eth-bytecode-db/eth-bytecode-db/src/verification/db.rs

```diff
@@ -15,6 +15,47 @@ use sea_orm::{
 };
 use std::collections::{BTreeMap, BTreeSet};
 
+macro_rules! insert_then_select {
+    ( $txn:expr, $entity_module:ident, $active_model:expr, [ $( ($column:ident, $value:expr) ),+ $(,)? ] ) => {
```

Comment:
I think? this can be done without macro
https://docs.rs/sea-orm/latest/sea_orm/entity/trait.EntityTrait.html
but it will probably require to redo this whole thing, so it's okay for now

Reply:
Would it be possible to use filters in that case?